### DISCUSSION TITLE
Open window in maximized mode

### DISCUSSION THREAD

**Discussion by bublikOff** [webui-dev/webui#465] [Q&A]
Can I open window in maximized mode? and disable to change window's size?

**Comment by wwderw** [comment]
You can in kiosk mode.

**Reply by bublikOff** [reply]
Its a bit different what I need .. I need to set minimal size of window as I need to display web GUI that requires minimal width and height of window to display UI "correctly"

**Reply by AlbertShown** [reply]
You can use webui webview window, get PID, then performe a Win32 APIs to maximaze it... but the best way, is probably create a simple window using any traditional GUI library, then add `webui.js` to your UI (_that's mean using webui only to connect UI to your backend_).
